dts/bindings/dma/xilinx,axi-dma-base.yaml

```diff
@@ -0,0 +1,58 @@
+# Copyright 2024 CISPA Helmholtz Center for Information Security gGmbH
+# SPDX-License-Identifier: Apache-2.0
+
+description: Xilinx AXI DMA LogiCORE IP controller
+
+include: dma-controller.yaml
+
+# multiple "compatible" properties match the same driver and options
+
+properties:
+  reg:
+    type: array
+    description: DMA Control registers
+    required: true
+
+  interrupts:
+    type: array
+    description: TX IRQ number followed by RX IRQ number
+    required: true
+
+  interrupt-parent:
+    type: phandle
+    description: Interrupt controller that the DMA is connected to
+
+  clocks:
+    type: phandle-array
+
+  clock-frequency:
+    type: int
+
+  xlnx,addrwidth:
+    type: int
+    default: 32
+    description: DMA address width (64 or 32 bit), defaults to 32 bit
```

Comment:
why does it default to 32 bit? should it just be required: true?

Reply:
Good point, I have changed this accordingly.